aten/src/ATen/native/vulkan/api/Command.cpp

```diff
@@ -242,17 +242,21 @@ void Command::Buffer::copy(
 }
 
 void Command::Buffer::dispatch(
-    const Shader::WorkGroup& work_group) {
+    const Shader::WorkGroup& global_work_group) {
   TORCH_INTERNAL_ASSERT_DEBUG_ONLY(
       command_buffer_,
       "This command buffer is in an invalid state! "
       "Potential reason: This command buffer is moved from.");
 
+  const auto div_round_up = [](const uint32_t n, const uint32_t d) {
```

Comment:
could it be static to create it only once?

Reply:
Great point, will do.